pkg/indexstorage/redis/redis.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package redis
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	redis "github.com/redis/go-redis/v9"
+)
+
```

Comment:
A little overkill, but do you want to add some tests? See https://github.com/sigstore/rekor/blob/75774ff1461ba5c2092febc8d340461722e01433/pkg/witness/publish_checkpoint_test.go#L65-L81 for using the redis mocks.